source/_integrations/uptimerobot.markdown

```diff
@@ -25,4 +25,8 @@ The **UptimeRobot** {% term integration %} provides entities to get the status f
 
 To get your API key, go to [My Settings](https://uptimerobot.com/dashboard#mySettings) on the UptimeRobot website, at the bottom you will find your "Main API Key".
 
+> **NOTE**  
```

Comment:
This is GitHub-specific syntax and won't work (see also the linked preview of this PR by Netlify).

Reply:
sorry about this, jinja2 include tag in the file itself should have been hint enough 🤦 
block removed anyway..